getting-started-github-graphql-api.md

```diff
@@ -23,15 +23,17 @@ summary = "Everything you need to get started with GitHub's GraphQL API for lear
+Step-by-step is often the easiest way learn something new, so it's fortunate that learning GraphQL in JavaScript teaches both the client and the server-side of an application. Seeing both sides of the web transactions is useful, but the catch is you have to learn two environments. The step-by-step mentality can be difficult to apply here, so I encourage newcomers to start client-side, by consuming a third-party GraphQL API, before the server side, which is using a GraphQL server. 
 
-In the following sections, I want to give you everything you need to get started with GitHub's GraphQL API for learning GraphQL in JavaScript from a client-side perspective first. After these sections, you should have a good clue about GitHub's terminology and you should have your GitHub account set up with data to be consumed later from the GraphQL API. Maybe not all of the following sections are applicable to you, because you may already have an active GitHub account and you might be prepared for these topics. There are a few more applications you are going to implement with GitHub's GraphQL API, so it makes lots of sense to invest the necessary time in this section to get everything up and running.
+{{% a_blank "GitHub" "https://github.com" %}} is one of the first major tech brands to adopt GraphQL.  They even managed to {{% a_blank "release" "https://githubengineering.com/the-github-graphql-api" %}} a public GraphQL API {{% a_blank "(official documentation)" "https://developer.github.com/v4" %}}, which is quite popular among developers, because most are familiar enough with GitHub from using it for their own projects.
+
+In this chapter, I hope to cover everything you need to get started with GitHub's GraphQL API, and learning to use GraphQL in JavaScript from a client-side perspective.  You should gain understanding about GitHub's terminology, and how to consume account data using its GraphQL API. There are a few more applications we will implement with the GraphQL API, so it makes sense to invest time into this section to avoid any fundamental mistakes.
```

Comment:
double spaces

Reply:
"In this chapter, I hope to cover everything you need to get started with GitHub's GraphQL API, and learning to use GraphQL in JavaScript from a client-side perspective." to "In this chapter, I hope to cover everything you need to get started with GitHub's GraphQL API to learn about GraphQL in JavaScript from a client-side perspective."

Reply:
"There are a few more applications we will implement with the GraphQL API" to ""There are a few more applications we will implement with Github's GraphQL API""